app/src/app/layout.js

```diff
@@ -22,8 +22,10 @@ export default function RootLayout({ children }) {
           className={`${openSans.variable} ${poppins.variable} ${quicksand.variable} ${ropaSans.variable} ${playfairDisplay.variable}`}
         >
           <NavBar />
-          {children}
-          <Footer />
+          <section> {children}</section>
+          <footer>
```

Comment:
why double footer?

Reply:
its not double footer
the <footer> tag are semantic tags providing a clear structure to both users and search engines
the <Footer> is a react component